### PR TITLE
added get and post routes for reviews

### PR DESCRIPTION
**Pull request by selinaduran**

### CODE REVIEW

File: components/ReviewModal.js [hunk 1]
```diff
@@ -1,24 +1,78 @@
...
+      });
+    } else {
+      temp.push(e.target.name);
+      this.setState({
+        categories: temp
+      })
+    }
+  }
+  handleSubmit = () => {
+    axios.post('http://localhost:8080/addreview', this.state)
```
Comment (tanner-white) [inline components/ReviewModal.js:50]: Did you test these routes with the database already?

Comment (selinaduran) [reply components/ReviewModal.js:50]: Yes! Jacob had helped me with making the routes, so we crosschecked with the database to make sure data was able to be successfully sent/received.

**Review by tanner-white — APPROVED** [review]

**Review by APolk10 — COMMENTED** [review]
@selinaduran this looks good. What was your reason for using a class component for the ReviewModal.js instead of using hooks.

**Comment by selinaduran** [comment]
> 

@Jacob-Gramer was the one that actually made the class component for ReviewModal, but I still think it's a great idea since the component is holding state for the form data that is submitted!

**Review by APolk10 — APPROVED** [review]

**Pull request merged**